validphys2/src/validphys/arclength.py

```diff
@@ -91,8 +121,9 @@ def plot_arc_lengths(
         ]
         yvalues = arclengths.stats.central_value()
         ylower, yupper = arclengths.stats.errorbar68()
-        ylower = yvalues - ylower
-        yupper = yupper - yvalues
+        ylower = np.abs(yvalues - ylower)
+        yupper = np.abs(yupper - yvalues)
```

Comment:
```suggestion
        ylower = np.maximum(yvalues - ylower, 0)
        yupper = np.maximum(yupper - yvalues, 0)
```

I think this is more consistent with what we do in other cases of negative numbers that we don't like (it should not matter to much though)

Reply:
As said above, I think the line should be always of length `yupper - ylower`. We could even remove the point in the middle and be done with it.

Reply:
That would also work for me.

Reply:
> As said above, I think the line should be always of length `yupper - ylower`

This is what the modification already does I think.
@Zaharid, @scarlehoff  I pushed the rebased version, but it's not running the tests.

Reply:
I think it was just that github needs a few minutes to reset (and the test failed because of the regression test since the AL has changed).

> This is what the modification already does I think.

Maybe I misunderstood @Zaharid but I think what he would like is just to plot `ylower`-----`yupper` instead of  `yvalues +/- (yupper, ywloer)`

Reply:
Except that yvalues should be something like (yupper - ylower)/2. Otherwise we are back to the original problem.